master: add table routing feature for cdc (#23211)
pingcap/docs: ticdc/ticdc-changefeed-config.md

#### `dispatchers`

> - In versions earlier than v8.5.7, you can use `dispatchers` to configure event dispatchers, and `dispatchers` takes effect only when the downstream is an MQ sink. Starting from v8.5.7, when you use the [new TiCDC architecture](/ticdc/ticdc-architecture.md), you can also use `dispatchers` to configure table routing, mapping upstream tables to specific downstream database or table names. For more information, see [TiCDC table routing](/ticdc/ticdc-table-routing.md).

Author: ```suggestion
- When the changefeed downstream is an MQ sink, you can use `dispatchers` to configure event dispatchers. Starting from v8.5.7, for the [new TiCDC architecture](/ticdc/ticdc-architecture.md), you can also use `dispatchers` to configure table routing, mapping upstream tables to specific downstream database or table names. For more information, see [TiCDC table routing](/ticdc/ticdc-table-routing.md).
```
